gradle.properties

```diff
@@ -4,3 +4,8 @@ untilBuildVersion=202.*
 #
 pluginGroup=zd-zero
 pluginVersion=1.2
+
+# This property allows you
+# to run/develop the plugin on a different IDE
+# ex: idePath=/home/alex/.local/share/JetBrains/Toolbox/apps/WebStorm/ch-0/202.6250.10
```

Comment:
minor: Don't want to be nitpicky but this hits my OCD lol
"ex" is not really an abbreviation, request to change it to "e.g."

Reply:
in my head it was short for `example` 😄 

![Small fixes](https://media.giphy.com/media/OlfmxuWHCmw2Q/giphy.gif)